src/backend/distributed/citus--7.0-15--7.0-16.sql

```diff
@@ -0,0 +1,24 @@
+/* citus--7.0-15--7.0-16 */
+
+SET search_path = 'pg_catalog';
+
+CREATE OR REPLACE FUNCTION master_move_node(node_name text,
+                                            node_port int,
```

Comment:
It'd be nicer to just use the node ID, since that makes the function idempotent and allows it to be called without knowledge of the current nodename and nodeport. That makes it easier to integrate in an external failover solution.

Reply:
Okay, good points.